src/DIRAC/Resources/Cloud/cloudinit.template

```diff
@@ -123,12 +124,14 @@ write_files:
        chmod +x dirac-install.py
      fi
 
-     python dirac-install.py -r %(release-version)s
+     curl -LO https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-x86_64.sh
```

Comment:
You should first try getting the installer from CVMFS.

In general, what's in https://github.com/DIRACGrid/Pilot/blob/master/Pilot/pilotCommands.py#L246 should be largely used here too.